src/sensai/featuregen.py

```diff
@@ -86,6 +103,11 @@ def generate(self, df: pd.DataFrame, ctx=None) -> pd.DataFrame:
                 if series.dtype.name != 'category':
                     resultDF[colName] = series.astype('category', copy=False)
 
+        self.generatedColumnNames = resultDF.columns
+        if self.generatedColumnsRuleTemplate is not None:
+            self._generatedColumnsRule = data_transformation.DFTNormalisation.Rule(r"(%s)" % "|".join(self.generatedColumnNames), **self.generatedColumnsRuleTemplate.__dict__)
```

Comment:
The names must be escaped with re.escape

Reply:
Done. This was also relevant in other places, I have written a helper function in util.string